ccd-definition/CaseEventToFields/DefendantResponseLRspec.json

```diff
@@ -539,7 +539,7 @@
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "PageID": "RequestedCourtLocationLRspec",
-    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"FULL_DEFENCE\" AND responseClaimTrack= \"SMALL_CLAIM\"",
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"FULL_DEFENCE\"",
```

Comment:
Looks like this screen is to be shown in both Fast track and small claim, are we sure we want to remove this small claim condition from here?

Reply:
Sorry ignore my comment. Indu explained it to me.

Reply:
Ok